### PR TITLE
[Backport - v2.0.0] TACKLE-659: Ensure selected persona will get cleared on logout

### PR DESCRIPTION
**Pull request by mturley**
Backports #288 (for https://issues.redhat.com/browse/TACKLE-659)

### CODE REVIEW

**Comment by mturley** [comment]
Note: do not merge until approved for 2.0.0 (under discussion on slack)

**Review by jortel — CHANGES_REQUESTED** [review]
HOLD pending issue triage.

**Review by gildub — APPROVED** [review]

**Review by jortel — APPROVED** [review]
LGTM
Issue triaged as blocker.

**Pull request merged**
